src/colvarmodule.cpp

```diff
@@ -1943,105 +1943,106 @@ template<typename T> std::string _to_str_vector(std::vector<T> const &x,
 }
 
 
-std::string cvm::to_str(int const &x,
-                        size_t width, size_t prec)
+
+std::string colvarmodule::to_str(int const &x,
```

Comment:
The difference between cvm:: and colvarmodule:: is unclear to me. In this commit a number of instances are converted to the second version, but in other files many cases of cvm:: were added. What is at stake here, besides brevity?

Reply:
Generally, I preferred to use the full `colvarmodule::` for the implementation of the class, and `cvm::` for the calls to its functions.  The difference is 100% cosmetic right now, but it would simplify moving to the use of a namespace if we need or want to in the future.